docroot/sites/default/settings.acquia.php

```diff
@@ -104,12 +112,17 @@
       $config['stage_file_proxy.settings']['origin'] = FALSE;
       $config['media_entity_download.settings']['external_file_storage'] = 0;
       $settings['mass_caching.hosts'] = ['edit.mass.gov', 'www.mass.gov'];
+
+      $config['akamai.settings']['disabled'] = FALSE;
+      $config['akamai.settings']['basepath'] = 'www.mass.gov';
```

Comment:
@weitzman in the  `akamai.setting.yml` file is see that the protocol is also included in the url `basepath: 'https://stage.mass.gov'` . Should we change it here as well?

Reply:
Yes, good catch. Fixed.